data/tutorials/gs_03_ocaml_on_windows.md

```diff
@@ -39,18 +112,50 @@ opam repo add upstream https://opam.ocaml.org --rank 2 --all-switches --set-defa
 This assumes you only have the `opam-repository-mingw` repository for this switch set with
 a priority of `1`.
 
-## opam.2.2.0
+### WSL2
+
+If you only need to _run_ OCaml programs on a Windows machine, then the simplest solution is to use the Windows Subsystem for Linux 2 (WSL2). WSL2 is a feature that allows Linux programs to run directly on Windows. WSL2 is substantially easier and faster to use than WSL1. Microsoft has comprehensive installation steps for [setting up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
 
-After the [successful release](https://github.com/ocaml/opam/releases/tag/2.1.0) 
-of opam 2.1.0 the [next version](https://github.com/ocaml/opam/projects/2) of
-`opam` will focus on closing the gap to fully supporting Windows. This includes
-supporting external dependency installation for Windows and integrating with the
-Windows shell. From an `opam-repository` perspective, the `ocaml-base-compiler`
-packages will need to support the mingw-w64 and MSVC variants.
+After you have installed WSL2 and chosen one Linux distribution (we suggest [Ubuntu LTS](https://apps.microsoft.com/store/detail/ubuntu/9PDXGNCFSCZV?hl=en-us&gl=US)), you can follow the
+[Get Up and Running With OCaml: Installation for Linux and macOS](/docs/up-and-running) steps.
 
-## Docker Images
+### Docker Images
 
 The [`ocaml/opam`](https://hub.docker.com/r/ocaml/opam) Docker Hub repository
 now contains regularly updated Windows images. This includes images using
 `msvc` and `mingw`. If you are comfortable with Docker, this might be an
 easier way to get a working Windows environment on your machine.
+
+## Editor Support for OCaml on Windows
+
+### Visual Studio Code on Windows
+
+**If you use the recommended DKML installer** you will need to:
+1. Go to `File` > `Preferences` > `Settings` view (or press `Ctrl ,`)
+2. Select `User` > `Extensions` > `OCaml Platform`
+3. **Uncheck** `OCaml: Use OCaml Env`
+
+**If you use WSL2** you will remotely connect to your WSL2 instance from
+Visual Studio Code. Microsoft has a [useful blog post](https://code.visualstudio.com/blogs/2019/09/03/wsl2)
+covering getting WSL2 and Visual Studio Code connected.
```

Comment:
```suggestion
that covers getting WSL2 and Visual Studio Code connected.
```

Just to avoid to -ing words next to each other, which has a tendency to trip up the flow